render/detailed/summary.go

```diff
@@ -45,19 +45,19 @@ type Column struct {
 
 // NodeSummary is summary information about a child for a Node.
 type NodeSummary struct {
-	ID         string               `json:"id"`
-	Label      string               `json:"label"`
-	LabelMinor string               `json:"labelMinor"`
-	Rank       string               `json:"rank"`
-	Shape      string               `json:"shape,omitempty"`
-	Stack      bool                 `json:"stack,omitempty"`
-	Linkable   bool                 `json:"linkable,omitempty"` // Whether this node can be linked-to
-	Pseudo     bool                 `json:"pseudo,omitempty"`
-	Metadata   []report.MetadataRow `json:"metadata,omitempty"`
-	Parents    []Parent             `json:"parents,omitempty"`
-	Metrics    []report.MetricRow   `json:"metrics,omitempty"`
-	Tables     []report.Table       `json:"tables,omitempty"`
-	Adjacency  report.IDList        `json:"adjacency,omitempty"`
+	ID            string                `json:"id"`
+	Label         string                `json:"label"`
+	LabelMinor    string                `json:"labelMinor"`
+	Rank          string                `json:"rank"`
+	Shape         string                `json:"shape,omitempty"`
+	Stack         bool                  `json:"stack,omitempty"`
+	Linkable      bool                  `json:"linkable,omitempty"` // Whether this node can be linked-to
+	Pseudo        bool                  `json:"pseudo,omitempty"`
+	Metadata      []report.MetadataRow  `json:"metadata,omitempty"`
+	Parents       []Parent              `json:"parents,omitempty"`
+	Metrics       []report.MetricRow    `json:"metrics,omitempty"`
+	PropertyLists []report.PropertyList `json:"propertyLists,omitempty"`
```

Comment:
I don't think this will break backwards compatibility (IFAICR this is only used between app<->UI) but worth double checking.